homeassistant/components/api_streams/sensor.py

```diff
@@ -4,65 +4,23 @@
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/sensor.api_streams/
```

Comment:
This sensor needs to be part of websocket_api. There is no reason for it to be its own integration ?

Reply:
Done, just needed to get rid of that comment.